add stableapi connection example (#590)
mongodb/docs-node: source/fundamentals/connection/connect.txt

   To learn how to retrieve your connection string in Atlas, see the
   :atlas:`Atlas driver connection guide </driver-connection>`.

The next part of the connection string contains your credentials
if you are using password-based authentication. Replace the value of ``user``
with your username and ``pass`` with your password. If you are using an
authentication mechanism that does not require a username and password, omit
this part of the connection URI.

> The next part of the connection string specifies the hostname or IP address,
> followed by the port of your MongoDB instance. In the example above, we use

Reviewer: I: This is a good change, but it introduces a new (possible) ambiguity: Does my MongoDB instance have a hostname/IP address as well as a port? Or does it just have a port?

S: 
```suggestion
The next part of the connection string specifies the hostname or IP address
of your MongoDB instance, followed by the port number. In the example above, we use
```

Happy to workshop this

.. _connect-sample-node-driver:

Atlas Connection Example
------------------------

> To connect to a MongoDB deployment on Atlas you must first create a client. You
> can create a client that uses your connection string and any other
> client options by passing your URI and a ``MongoClientOptions`` object to a
> ``MongoClient``.

Reviewer: Suggest something like the following for brevity and clarity:

```suggestion
You must create a client to connect to a MongoDB deployment on Atlas.
To create a client, construct an instance of ``MongoClient``, passing in your 
URI and a ``MongoClientOptions`` object.
```

> You can set the Stable API version client option to avoid breaking changes when

Reviewer: Confusing that the option is called `serverApi` in code :/

I wonder if this could be phrased more directly; instead of telling people why we're using Stable API (avoiding breaking changes), we can just tell them what this option is doing:

"The ``serverApi`` option in ``MongoClientOptions`` enables the Stable API feature, which forces the server to run operations with behavior compatible with the specified API version. [skip the Stable API link] The following code shows how..."

[Stable API link in admonition at bottom]

Again, happy to workshop!

Author: Changed the wording slightly from your suggestion, but wrote something pretty similar. Let me know if you feel that's a good middle ground or not
